cucumber/api/features/authenticate.feature

```diff
@@ -20,6 +20,11 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticating with no Content-Type header succeeds without leakage.
```

Comment:
Minor but we don't usually punctuate single sentence descripts (i.e. no period). Also could we say what exactly isn't being leaked and where it's not getting leaked to? So something like "succeeds without writing API key to the logs".

Reply:
Done.  Unfortunately Gherkin has no way to break up long lines, but we clarity is more important than line overflow.